version_mapping/3.0.0-3.1.0/UpgradeShapes-3.0.0-to-3.1.0.ttl

```diff
@@ -130,15 +132,15 @@ ids:MessageUpgradeShape
 			$this ids:contractRejectionReason ?newContractRejectionReason .
 			idsm:VersionUpgrade ids:contractRejectionReason ?oldContractRejectionReason.
 			
-			$this ids:issued ?newIssued .
-			idsm:VersionUpgrade ids:issued ?oldIssued.
+			$this dct:issued ?newIssued .
```

Comment:
These things in the `version_mapping` directory must not be updated! They represent contents of older IM versions, and therefore must stay as they are. Please revert the "ids to dct/dcat" upgrades in both 3.0.0->3.1.0 and 3.1.0->4.0.0, thanks!